include/pros/adi.h

```diff
@@ -69,6 +69,11 @@ typedef enum adi_port_config_e {
 	E_ADI_ERR = PROS_ERR
 } adi_port_config_e_t;
 
+typedef enum adi_potentiometer_type_e { 
+	OLD_POTENTIOMETER = false,
+	NEW_POTENTIOMETER = true
+} adi_potentiometer_type_e_t;
+
```

Comment:
```suggestion
typedef enum adi_potentiometer_type_e { 
	E_ADI_POT_EDR = 0,
	E_ADI_POT_V2
} adi_potentiometer_type_e_t;

```

This enum does not follow how the others are styled, and we shouldn't keep these as booleans in case more potentiometer types are released by vex.